### ISSUE TITLE
train loss

### ISSUE THREAD

**Issue by emiyashiroz** [marco-rudolph/differnet#42]
Is it normal for the training loss to be less than 0?

**Comment by marco-rudolph** [comment]
Yes. The loss reflects the negative log likelihood that can be less than zero.
